product_search/product_search.py

```diff
@@ -0,0 +1,46 @@
+"""Program that returns inventory details based for an inventory item"""
+
+import sys
+from data_management import access_json_data
+
+data = access_json_data(sys.argv[1])
+
+
+def get_product_name():
+    """Get a name of a product from the user"""
+    return input("What is the product name? ").title()
+
+
+def item_inventory_details(item_name):
+    """Verify that the product exists; if no record is found, prompt again."""
+
+    while True:
+        for item in data["products"]:
+            if item["name"] == item_name:
+                return item
+        print("Sorry, that product was not found in our inventory.")
+        item_name = get_product_name()
+
+
+def _style_dictionary_items_for_output(inv_details):
+    """Update keys, values in inventory dictionary for user readability"""
+    user_inventory_items = {}
+
+    for category, details in inv_details.items():
+        if category == "quantity":
+            category = "Quantity on hand"
+        if category == "price":
+            details = "${0:.2f}".format(inv_details["price"])
+        user_inventory_items[category] = details
+    return user_inventory_items
+
+
+def display_product_information(inv_details):
+    """Return the product information associated with an item"""
+
+    inv_details = _style_dictionary_items_for_output(inv_details)
+
+    report_details = ""
+    for category, details in inv_details.items():
+        report_details += f"{category.capitalize()}: {details}\n"
+    print(report_details)
```

Comment:
Consider returning `report_details` and then allowing `main` to determine how to output that to the user. Remember `main` should be responsible for input and output. Not your core functionality.